client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-methods/utils.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const getMethodSummary = ( method, currencyPrefix = '', currencySuffix = '' ) => {
+	switch ( method.methodType ) {
+		case 'flat_rate':
+			return translate( 'Cost: %(prefix)s%(cost)s%(suffix)s', { args: {
+				prefix: currencyPrefix,
+				cost: method.cost,
+				suffix: currencySuffix,
+			} } );
+		case 'free_shipping':
+			if ( ! method.requires ) {
+				return translate( 'For everyone' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Everyone', { context: 'Filter label for posts list'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/14275-shipping-methods-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).